cargo-shuttle/src/config.rs

```diff
@@ -122,17 +122,17 @@ impl ConfigManager for LocalConfigManager {
 /// Global client config for things like API keys.
 #[derive(Deserialize, Serialize, Default)]
 pub struct GlobalConfig {
-    pub api_key: Option<ApiKey>,
+    api_key: Option<String>,
```

Comment:
Why is this needed?

Reply:
So the ApiKey was just a type alias for `String`. By also making it not pub you can't set it (unless you manually edit the config file) without using the `set_api_key` method, which takes an `ApiKey` (the newtype that can only be created by `ApiKey::parse`, so it will always be a valid format).